src/modules/container.rs

```diff
@@ -7,11 +7,22 @@ pub fn module<'a>(_context: &'a Context) -> Option<Module<'a>> {
 
 #[cfg(target_os = "linux")]
 pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    use std::fs::File;
+    use std::io::Read;
+
     use super::ModuleConfig;
     use crate::configs::container::ContainerConfig;
     use crate::formatter::StringFormatter;
     use crate::utils::read_file;
 
+    fn check_if_docker_in_cgroup() -> Option<bool> {
+        let mut cgroup_file = File::open("/proc/1/cgroup").ok()?;
+        let mut cgroup_content = String::new();
+        cgroup_file.read_to_string(&mut cgroup_content).ok()?;
```

Comment:
You could use the helper `crate::utils::read_file` to shorten this a bit.
Beyond this, please use `context_path` for the absolute `/proc/1/cgroup` path (documented in `Contributing.md` and also used mainly in this file) and please use it to write a test for this.